code/game/objects/items/charter.dm

```diff
@@ -93,3 +95,5 @@
 
 	if(!unlimited_uses)
 		used = TRUE
+
+#undef STATION_RENAME_TIME_LIMIT
```

Comment:
Why undefine it if you can use this for the build your own shuttle buy time as well?